modules/nodes-pods-autoscaling-custom-rn.adoc

```diff
@@ -26,23 +26,37 @@ The following table defines the Custom Metrics Autoscaler Operator versions for
 |{product-title} version
 |General availability
 
-|2.8.2
-|4.13
-|Technology Preview
-
-|2.8.2
+|2.8.2-174
 |4.12
 |Technology Preview
 
-|2.8.2
+|2.8.2-174
 |4.11
 |Technology Preview
 
-|2.8.2
+|2.8.2-174
 |4.10
 |Technology Preview
 |===
 
+[id="nodes-pods-autoscaling-custom-rn-282-174_{context}"]
+== Custom Metrics Autoscaler Operator 2.8.2-174 release notes
+
+This release of the Custom Metrics Autoscaler Operator 2.8.2-174 provides new features and bug fixes for running the Operator in an {product-title} cluster. The components of the Custom Metrics Autoscaler Operator 2.8.2-174 were released in link:https://access.redhat.com/errata/RHEA-2023:1683[RHEA-2023:1683].
```

Comment:
`were released` Is this past tense intentional? I checked Errata and couldn't find this advisory. Or am I misunderstanding something here?

Reply:
@snarayan-redhat I took that wording from the [WMCO release notes](https://docs.openshift.com/container-platform/4.12/windows_containers/windows-containers-release-notes-7-x.html#wmco-7-0-1).

As for the link, I need to find a better way to state this. The errata is not live until the new version is released. When the errata is released, I check the link before merging.